src/sympc/protocol/falcon/falcon.py

```diff
@@ -18,6 +18,21 @@ class Falcon(metaclass=Protocol):
 
     """Used for Share Level static operations like distrubuting the shares."""
     share_class: SyMPCTensor = ReplicatedSharedTensor
+    security_levels: List[str] = ["semi-honest", "malicious"]
+
+    def __init__(self, security_type: str = "semi-honest"):
+        """Intializer for the Protocol.
+
+        Args:
+            security_type : specifies the security level of the Protocol.
+
+        Raises:
+            ValueError : If invalid security_type is provided.
+        """
+        if security_type not in self.security_levels:
+            raise ValueError(f"{security_type} is not a valid securtiy type")
```

Comment:
small typos in "Intializer" "securtiy" -> you might want to integrate a spell check in your code editor, it will highlight this for you :)